gbpservice/nfp/orchestrator/modules/device_orchestrator.py

```diff
@@ -162,12 +173,36 @@ def network_function_notification(self, context, notification_data):
         request_info['network_function_device_id'] = nfd_id
         event_data = request_info
         event_data['id'] = request_info['network_function_device_id']
-        self._create_event(event_id=event_id,
-                           event_data=event_data)
-        nfp_logging.clear_logging_context()
+        if event_id == "HEALTH_MONITOR_COMPLETE" or event_id == 'DEVICE_NOT_REACHABLE':
+            event = self._controller.new_event(id="DEVICE_UP",
+                                               key=nf_id)
+            event_desc = nfp_context.pop('event_desc')
+            event = self._set_event_desc_to_event(
+                event,
+                event_desc)
+
+            if result.lower() == "success":
+                self._controller.event_complete(event, result="SUCCESS")
```

Comment:
In the same function above we are already checking for result, can we use that, instead of adding 1 more check